src/openforms/submissions/tests/test_task_co_sign.py

```diff
@@ -97,6 +98,48 @@ def test_happy_flow(self):
 
         self.assertEqual(email.recipients(), ["test@test.nl"])
 
+    def test_form_link_allowed_in_email(self):
+        submission = SubmissionFactory.from_components(
+            components_list=[
+                {
+                    "key": "cosign",
+                    "type": "cosign",
+                    "label": "Cosign component",
+                },
+            ],
+            submitted_data={"cosign": "test@test.nl"},
+        )
+
+        send_email_cosigner(submission.id)
```

Comment:
Try to patch the call to get the configuration, I think this is the problem for the reverse test suite.
(and do the same for the test below)